nexmo/__init__.py

```diff
@@ -154,6 +172,50 @@ def get_number_insight(self, params=None, **kwargs):
   def request_number_insight(self, params=None, **kwargs):
     return self.post(self.host, '/ni/json', params or kwargs)
 
+  def get_applications(self, params=None, **kwargs):
+    return self.get(self.api_host, '/v1/applications', params or kwargs)
+
+  def get_application(self, application_id):
+    return self.get(self.api_host, '/v1/applications/' + application_id)
+
+  def create_application(self, params=None, **kwargs):
+    return self.post(self.api_host, '/v1/applications', params or kwargs)
+
+  def update_application(self, application_id, params=None, **kwargs):
+    return self.put(self.api_host, '/v1/applications/' + application_id, params or kwargs)
+
+  def delete_application(self, application_id):
+    return self.delete(self.api_host, '/v1/applications/' + application_id)
+
+  def start_call(self, params=None, **kwargs):
```

Comment:
`start_call` -> `create_call`?

Consistent with PHP and Node libraries. And consistent with `create_application`.